src/components/Nikto/Nikto.tsx

```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import { Button, Checkbox, LoadingOverlay, Stack, TextInput } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
+import { UserGuide } from "../UserGuide/UserGuide";
+
+// Define title and user guide description for the Nikto tool
+const title = "Nikto";
+const description_userguide =
```

Comment:
- [ ] Seperate into guide and description variables required for render component